src/randoop/main/GenTests.java

```diff
@@ -429,264 +399,335 @@ public boolean handle(String[] args) throws RandoopTextuiException {
         }
       }
     }
+    
+    ExecutionVisitor visitor;
+    if (visitors.isEmpty()) {
+      visitor = new DummyVisitor();
+    } else {
+      visitor = new MultiVisitor(visitors);
+    }
+    
+    explorer.addTestPredicate(outputTest);
+    explorer.addTestCheckGenerator(testGen);
+    explorer.addExecutionVisitor(visitor);
 
-    explorer.executionVisitor.visitors.addAll(visitors);
-
+    if (!GenInputsAbstract.noprogressdisplay) {
+      System.out.printf ("Explorer = %s\n", explorer);
+    }
+    
+    //Generate tests
```

Comment:
When you make a change on a line, you remove the code review conversation 
related to that line.  I believe you made a whitespace change on this 
line, so I don't know the referent for "what you are referring to here".  
Unfortunately, that means I can't respond to your comment.